include/phasar/PhasarLLVM/ControlFlow/LLVMBasedICFG.h

```diff
@@ -107,7 +114,10 @@ class LLVMBasedICFG
   /// Maps functions to the corresponding vertex id.
   std::unordered_map<const llvm::Function *, vertex_t> FunctionVertexMap;
 
-  void constructionWalker(const llvm::Function *F, Resolver &Resolver);
+  void processFunction(const llvm::Function *F, Resolver &Resolver,
+                           bool &FixpointReached);
```

Comment:
Formatting ?

Reply:
@pdschubert will do this at the time of merge. There is other unrelated stuff where formatting needs to be done.